### DISCUSSION TITLE
issue: (probably not caused by skia-safe) Segmentation fault while closing window

### DISCUSSION THREAD

**Discussion by marc2332** [rust-skia/rust-skia#862] [Q&A]
Original issue https://github.com/marc2332/freya/issues/347

Hey Armin, do you know what could have caused that crash? Could it be my fault? I have never seen it before

Thanks! 😄

**Comment by pragmatrix** [comment]
As Simon already noted, drop order could be an issue. We've had a similar issue on Windows a while ago: https://github.com/rust-skia/rust-skia/issues/476#issuecomment-856058912

**Reply by marc2332** [reply]
Looks like it [didn't](https://github.com/marc2332/freya/issues/347#issuecomment-1775713196) solve it, do you have any clue what other thing could be causing it?
